validation/src/test/scala/hmda/validation/rules/ts/validity/TsEditCheckSpec.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.rules.ts
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.parser.fi.ts.TsGenerators
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.EditCheck
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+abstract class TsEditCheckSpec extends PropSpec with PropertyChecks with MustMatchers with TsGenerators {
+  implicit val generatorDriverConfig =
+    PropertyCheckConfig(minSuccessful = 100, maxDiscarded = 500)
+
+  def check: EditCheck[TransmittalSheet]
+
+  // TODO consider trying Matcher instead of these methods
+  implicit class LarChecker(ts: TransmittalSheet) {
```

Comment:
I removed the commit from #346 that duplicates this PR, so no need to worry about conflicts.